src/app/child-dev-project/attendance/model/activity-attendance.ts

```diff
@@ -0,0 +1,150 @@
+import { Note } from "../../notes/model/note";
+import {
+  AttendanceCounting,
+  AttendanceStatus,
+  DEFAULT_ATTENDANCE_TYPES,
+} from "./attendance-status";
+import { Entity } from "../../../core/entity/entity";
+import { RecurringActivity } from "./recurring-activity";
+
+export class ActivityAttendance extends Entity {
```

Comment:
My current concept is to calculate these dynamically based on events from the database (no writing of duplicate information). In a second step, later on, we can add a variant of this that can be persisted to the db to allow users recording a monthly total rather than individual events.
https://drive.google.com/file/d/17ZF0UZwQ8TTwWxLiz5PNm4OLHm0M0MMk/view?usp=sharing

I'll add comments and documentation explaining this.